pkg/yurtadm/cmd/reset/reset_test.go

```diff
@@ -0,0 +1,327 @@
+/*
+Copyright 2021 The OpenYurt Authors.
```

Comment:
Please replace the year as 2022.

Reply:
Thanks for your suggestions，the bug is already fixed